From 0d086b22b48b23df141c3ca4b120f76537329e2e Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Fri, 20 Dec 2024 14:27:43 +0100
Subject: [PATCH 768/954] fix(stm32mp2-fdts): set SDMMC max frequency to 166
 MHz on stm32mp23

Set SDMMC max frequency to 166 MHz on stm32mp23.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git d7e803bee7e1e46474dbbc9e17346a4426c7ba13 fix(stm32mp2-fdts): set SDMMC max frequency to 166 MHz on stm32mp23]
Change-Id: Ia88fe8fdfce9f54d13cff8cafe581681bd036f6c
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/426153
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
---
 fdts/stm32mp231.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fdts/stm32mp231.dtsi b/fdts/stm32mp231.dtsi
index 3b0864d690..8a8a4af4f7 100644
--- a/fdts/stm32mp231.dtsi
+++ b/fdts/stm32mp231.dtsi
@@ -278,7 +278,7 @@
 				resets = <&rcc SDMMC1_R>;
 				cap-sd-highspeed;
 				cap-mmc-highspeed;
-				max-frequency = <120000000>;
+				max-frequency = <166000000>;
 				status = "disabled";
 			};
 
@@ -291,7 +291,7 @@
 				resets = <&rcc SDMMC2_R>;
 				cap-sd-highspeed;
 				cap-mmc-highspeed;
-				max-frequency = <120000000>;
+				max-frequency = <166000000>;
 				status = "disabled";
 			};
 		};
-- 
2.53.0

